Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=3e420e78ece6f9d2accc1568e80dfd0501e13df1
Commit:     3e420e78ece6f9d2accc1568e80dfd0501e13df1
Parent:     8317e0b0c2234f5f1f5d54804e4093d11bc0dffa
Author:     Luke Yelavich <themuso@ubuntu.com>
AuthorDate: Tue Dec 16 12:37:47 2008 +1100
Committer:  Takashi Iwai <tiwai@suse.de>
CommitDate: Wed Jan 14 08:08:49 2009 +0100

    ALSA: hda - Add automatic model setting for Samsung Q45
    
    Have the Samsung Q45 (144d:c510) select ALC262_HIPPO by default
    
    Reference: Ubuntu bug 200210
    	http://launchpad.net/bugs/200210
    
    Signed-off-by: Luke Yelavich <themuso@ubuntu.com>
    Cc: stable@kernel.org
    Signed-off-by: Takashi Iwai <tiwai@suse.de>

 sound/pci/hda/patch_realtek.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index ea4c88f..82dd084 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -10573,6 +10573,7 @@ static struct snd_pci_quirk alc262_cfg_tbl[] = {
 	SND_PCI_QUIRK(0x10cf, 0x142d, "Fujitsu Lifebook E8410", ALC262_FUJITSU),
 	SND_PCI_QUIRK(0x144d, 0xc032, "Samsung Q1 Ultra", ALC262_ULTRA),
 	SND_PCI_QUIRK(0x144d, 0xc039, "Samsung Q1U EL", ALC262_ULTRA),
+	SND_PCI_QUIRK(0x144d, 0xc510, "Samsung Q45", ALC262_HIPPO),
 	SND_PCI_QUIRK(0x17aa, 0x384e, "Lenovo 3000 y410", ALC262_LENOVO_3000),
 	SND_PCI_QUIRK(0x17ff, 0x0560, "Benq ED8", ALC262_BENQ_ED8),
 	SND_PCI_QUIRK(0x17ff, 0x058d, "Benq T31-16", ALC262_BENQ_T31),
